test/functional/feature_fastprune.py

```
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+# Copyright (c) 2023 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test fastprune mode."""
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+    assert_equal
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)